File: src/model/TrailerType.java
package model;

public enum TrailerType {
    // Measured in 'rulle-paller', if I remember correctly a EU-pal is 1.4 'rulle-palle'
    STOR (51.0);
    
    private final double capacity;
    
    TrailerType(double capacity) {
        this.capacity = capacity;
    }

    /**
     * @return the capacity
     */
    public double getCapacity() {
        return capacity;
    }    

}

File: Model/TrailerType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Model
{
    public class TrailerType
    {
        public const double STOR = 51.0;
        public double Capacity { get; private set; }
        
        public TrailerType(double capacity)
        {
            this.Capacity = capacity;
        }
    }
}


